### PR TITLE
Add Manufacturer data to device events and avoid duplicate date

### PR DESCRIPTION
**Pull request by Raffone17**
## 📑 Description

Closes # 

## ✅ Checks

- [X] My pull request adheres to the code style of this project
- [X] My code requires changes to the documentation
- [X] I have updated the documentation as required
- [X] All the tests have passed

### CODE REVIEW

**Review by arol-varesi — COMMENTED** [review]
Leave here my comment, waiting for explicit approval from @gmacario.
My comment is: 👍

**Review by gmacario — APPROVED** [review]
Apart from a minor typo in a commit log, LGTM.

**Pull request merged**
